### PR TITLE
Create server app.

### PR DESCRIPTION
**Pull request by VitalySokolov**

### CODE REVIEW

File: src/middlewares/cookieParser.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module.exports = (req, res, next) => {
+  const cookieString = req.headers.cookie;
+  const cookies = {};
+
+  if (cookieString) {
+    cookieString.split(';')
```
Comment (pavel-ulasavets) [inline src/middlewares/cookieParser.js:6]: Minor: Array.prototype.reduce might work well in this circumstances

File: src/routes/products.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
+const express = require('express');
+
+const router = express.Router();
+
+let nextId = 4;
+const products = [
```
Comment (pavel-ulasavets) [inline src/routes/products.js:6]: In general, it is not the best idea to mix data layer together with router logic. From my perspective it is much better to create a separate module which will be responsible for returning the data as well as data validation upon adding/removing entries

**Review by pavel-ulasavets — CHANGES_REQUESTED** [review]
In general the homework is done well (thank you for that btw:)) The only comment which I'd like to be fixed is a separation of data model operations and routing logic cause (and I believe you know it) these are two different responsibilities and it will be complicated as hell to maintain that code in future

File: src/middlewares/cookieParser.js (hunk 1 above)
Comment (VitalySokolov) [reply src/middlewares/cookieParser.js:6]: Thanks for tip. Will try to refactor this in the next PR.

File: src/routes/products.js (hunk 2 above)
Comment (VitalySokolov) [reply src/routes/products.js:6]: Absolutely agree. It was done with fake data only as example that it'll be working.
I already moved data and validation logic to the separate module in the next PR.

**Review by pavel-ulasavets — APPROVED** [review]

**Pull request merged**